components/gc-features/gc-features-en.html

```diff
@@ -49,17 +49,17 @@ <h2>Features</h2>
 	<div class="row wb-eqht">
 		<div class="col-lg-4 col-md-6 mrgn-bttm-md">
 			<h3 class="h5"><a href="#" class="stretched-link">[Feature hyperlink text]</a></h3>
-			<img src="{{assets}}/img/360x203.png" alt="" class="img-responsive thumbnail mrgn-bttm-sm">
+			<img src="../img/360x203.png" alt="" class="img-responsive thumbnail mrgn-bttm-sm">
```

Comment:
@duboisp I think we'll need to discuss this further at some point since I feel like we kind of went in circles around the merits of having a single unique high-level version of a file vs having several low-level copies.

Reply:
If it is used by other component, keep that image in the component/img.
Idem for template.

In general, images would need to be renamed by the purpose not by their technical size

Reply:
I've tried addressing this somewhat in my latest force pushes:
* Image file names that are named after resolutions are now prefixed with their purposes
  * Example: ``360x203.png`` -> ``feature-360x203.png``
* After determining image purposes and whether they're re-used across multiple pages, I moved them to what I felt to be appropriate folder locations
  * I tried looking at the [Canada.ca Design System (CDS)](https://design.canada.ca/) while establishing image purposes, but in the end opted to ignore it... Many of GCWeb's page templates deviate too much from the CDS to be able to rely on it for what I was doing.

**Note:**
All of this was only done for image file names that were affected by broken links. The repo still contains several other image files that are named after resolutions.